examples/cpp/SampleCpp/SampleCpp.vcxproj

```diff
@@ -87,39 +87,46 @@
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
     <CharacterSet>Unicode</CharacterSet>
+    <PlatformToolset>v142</PlatformToolset>

```

Comment:
why explicit retarget to v142 is required?

Reply:
Whoops good catch! This is left over from VS automatically retargeting the project. I'll revert this.